linux/vtpm_cold_add_remove/vtpm2_activatecredential_test.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+#
+- name: "Check supporting ActivateCredential or not"
```

Comment:
These multiple tasks can be put in a single shell script or in a single task with shell commands to avoid the whole testing disrupted by connection issue.